plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/ElasticsearchQueryRunner.java

```diff
@@ -45,6 +47,7 @@ private ElasticsearchQueryRunner() {}
 
     private static final Logger LOG = Logger.get(ElasticsearchQueryRunner.class);
     private static final String TPCH_SCHEMA = "tpch";
+    private static final boolean START_ELASTICSEARCH_SERVER = parseBoolean(System.getProperty("elasticsearch.test.start-server"));
```

Comment:
This should be a proper argument to the query runner, not a magic system property

Reply:
> a proper argument to the query runner"

Do you mean passing as `String[] args` in the main method?